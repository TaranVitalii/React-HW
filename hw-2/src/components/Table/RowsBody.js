import React from "react";

export default props => {
  return (
    <tr>
      <td>{props.name}</td>
      <td>{props.value}</td>
    </tr>
  );
};
