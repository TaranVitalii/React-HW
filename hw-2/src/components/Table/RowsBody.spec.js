import React from "react";
import RowsBody from "./RowsBody";
import { shallow } from "enzyme";

describe("<RowsBody/>", () => {
  it("should render a RowsBody", () => {
    const wrapper = shallow(<RowsBody />);

    expect(wrapper).toMatchSnapshot();
  });
  it("should render a document title", () => {
    let toBe = {
      children: [<td>festal</td>, <td>Jest</td>]
    };
    const wrapper = shallow(<RowsBody name="festal" value="Jest" />);
    expect(wrapper.props("name")).toStrictEqual(toBe);
  });
});
