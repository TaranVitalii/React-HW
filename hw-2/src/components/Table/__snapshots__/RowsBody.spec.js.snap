// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RowsBody/> should render a RowsBody 1`] = `ShallowWrapper {}`;
